.azure_pipelines/scripts/run-azure-tests.sh

```diff
@@ -9,12 +9,18 @@ az account show
 export AZURE_CLIENT_ID="${servicePrincipalId}"
 export AZURE_TENANT_ID="${tenantId}"
 export AZURE_CLIENT_SECRET="${servicePrincipalKey}"
-AZURE_SUBSCRIPTION_ID="$(az account show --query id --output tsv)"
-export AZURE_SUBSCRIPTION_ID
+export AZURE_SUBSCRIPTION_ID="$(az account show --query id --output tsv)"
```

Comment:
Split this to 2 lines for best practices.

Reply:
Didn't know about this, thank you for pointing it out! I learnt something